deploy/piraeus/charts/etcd/templates/service.yaml

```diff
@@ -0,0 +1,24 @@
+---
+# Source: piraeus/charts/etcd/templates/service.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+metadata:
+  name: piraeus-op-etcd
+  labels:
+    heritage: "Helm"
```

Comment:
I guess this means that we could manage this deployment with Helm :smile: But that's not a problem.